src/components/FeeLabelToggle/FeeLabelToggle.js

```diff
@@ -94,8 +94,8 @@ const FeeLabelToggle = ({
   const feeValue = getFormattedTransactionFeeValue(chain, txFeeInWei, gasToken);
   const currencySymbol = getCurrencySymbol(fiatCurrency);
 
-  const gasSymbol = getGasSymbol(chain, gasToken);
-  const feeInFiat = parseFloat(feeValue) * getRate(chainRates, gasSymbol, fiatCurrency);
+  const gasAddress = gasToken?.address || nativeAssetPerChain[chain].address;
```

Comment:
Maybe we could create some helper `function getGasAddress(gasToken, chain)` that would centralise it